flang/test/Fir/real.fir

```diff
@@ -21,18 +21,18 @@ func @bar(%a : !fir.real<2>, %b : !fir.real<4>, %c : !fir.real<8>, %d : !fir.rea
 }
 
 // CHECK-LABEL: @foo
-func @foo(%a : !fir.real<16>, %b : !fir.real<16>, %c : !fir.real<16>, %d : !fir.real<16>, %e : !fir.real<16>) -> !fir.real<16> {
+func @foo(%a : f16, %b : f16, %c : f16, %d : f16, %e : f16) -> f16 {
```

Comment:
!fir.real<16> is actually f128. i.e., quad precision.

Reply:
Thanks for explaining! These were bytes representation :) 2,4,8,10,16

Reply:
Almost, yes. Pedantically, they are KIND values. The KIND value of 3, in fact, maps to a 16-bit float representation. A Fortran front-end can use its own arbitrary convention.